src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -386,13 +393,16 @@ public MainPanel(MainMenu parentFrame) {
 				if (u.getMySubmenu().equals("main")) {
 					button[k] = new JButton(u.getButtonLabel());
 					button[k].setMnemonic(KeyEvent.VK_A + (u.getShortcut() - 'A'));
-
-					button[k].addActionListener(parentFrame);
+					if ("logout".equals(u.getCode())) {
+						addLogoutButtonListener(k);
+					} else {
+						button[k].addActionListener(parentFrame);
+					}
```

Comment:
I don't know if it is really needed a MENUITEM for the logout: the button should always appear when SINGLEUSER=no, and should not appear when SINGLEUSER=yes.
Having a MENUITEM entry make sense when we want eventually disable the logout function for some user, but we do?
Of course, having this entry simplify the button construction in the main menu...